[ZoneTelechargement] Add support for Streaming links

sysadminstory opened the pull request:
The bridge retrieves Streaming links now. It permits to list only
Direct Download Links, Streaming Links or both. The existing feeds will
show both ; this prevents to break the existing feeds.

To retrieve Streaming links, an additionnal HTTP request is necessary,
as the unprotected URIs shows only Streaming or Direct Download links/

sysadminstory committed:
[ZoneTelechargement] Add support for Streaming links

The bridge retrieves Streaming links now. It permits to list only
Direct Download Links, Streaming Links or both. The existing feeds will
show both ; this prevents to break the existing feeds.

To retrieve Streaming links, an additionnal HTTP request is necessary,
as the unprotected URIs shows only Streaming or Direct Download links/

em92 commented on bridges/ZoneTelechargementBridge.php:
```diff
@@ -18,57 +18,109 @@ class ZoneTelechargementBridge extends BridgeAbstract {
+				$item['title'] = $this->showTitle . ' Episode ' . $epnum . ' - Streaming';
+				// As RSS Bridge use the URI as GUID they need to be unique : adding a md5 hash of the title element
+				// should geneerate unique URI to prevent confusion for RSS readers
+				$item['uri'] = self::URI . $this->getInput('url') . '#' . hash('md5', $item['title']);
```
You can set it to `$item['uid']` and leave `$item['url']` as original without hash.

em92 commented on bridges/ZoneTelechargementBridge.php:
```diff
@@ -18,57 +18,109 @@ class ZoneTelechargementBridge extends BridgeAbstract {
+				$item['title'] = $this->showTitle . ' Episode ' . $epnum . ' - Téléchargement';
+				// As RSS Bridge use the URI as GUID they need to be unique : adding a md5 hash of the title element
+				// should geneerate unique URI to prevent confusion for RSS readers
+				$item['uri'] = self::URI . $this->getInput('url') . '#' . hash('md5', $item['title']);
```
Same as below

em92 replied:
*above

em92 reviewed (commented):
Hi, @sysadminstory !

sysadminstory committed:
[ZoneTelechargement] Set UID for each item

UID are now set with a hash of the item title for every item title, and
the link is the same for every item.

sysadminstory commented:
This commit follow your suggestion !

@em92 : Is still something missing ?

em92 commented:
Looks like I missed it my notification list
gj @sysadminstory !
